build/webpack.config.js

```diff
@@ -68,6 +62,11 @@ webpackConfig.output = {
 // ------------------------------------
 webpackConfig.plugins = [
   new webpack.DefinePlugin(config.compiler_globals),
+  new webpack.ProvidePlugin({
+    $: 'jquery',
+    jQuery: 'jquery',
+    'window.jQuery': 'jquery',
+  }),
```

Comment:
We provide jQuery so semantic jQuery plugins can find it.  This will soon be removed as we are dropping jQuery entirely